rocky/rocky/views/mixins.py

```diff
@@ -46,6 +48,19 @@ class OriginData(BaseModel):
     boefje: Boefje | None = None
     params: dict[str, str] | None = None
 
+    @property
+    def is_old(self, maxage: Optional[int] = None) -> bool
+        if not maxage:
```

Comment:
See suggestion above. But even if it was nullable, it would make more sense to explicitly check for `None` instead. Otherwise, it's ambiguous; are you checking for `None`, a value of `0`, or both?

Additionally, consider implementing a range check. For example, check if the value is greater than 0 and raise a `ValueError` otherwise